base_layer/wallet_ffi/src/lib.rs

```diff
@@ -1550,7 +1551,8 @@ pub unsafe extern "C" fn encrypted_value_as_bytes(
 #[no_mangle]
 pub unsafe extern "C" fn encrypted_value_destroy(encrypted_value: *mut TariEncryptedValue) {
     if !encrypted_value.is_null() {
-        Box::from_raw(encrypted_value);
+        // zeroize the data content of encrypted_value, as to prevent memory leaks
```

Comment:
Is this functionality even necessary since the encrypted value is already included on chain?